manifest.yaml

```diff
@@ -1,114 +1,109 @@
-marketing-site: "https://start9.com/"
-# The series of commands to build the project into an s9pk for arm64/v8. In this case we are using a Makefile with the simple build command "make".
+# TODO: This repo does not exist yet, but I think it's correct to assume that it will?
+wrapper-repo: "https://github.com/Start9Labs/itchysats-wrapper"
+upstream-repo: "https://github.com/itchysats/itchysats"
+support-site: "https://github.com/itchysats/itchysats/issues"
+marketing-site: "https://itchysats.network"
 build: ["make"]
-# Human readable descriptors for the service. These are used throughout the EmbassyOS user interface, primarily in the marketplace.
 description:
-  # This is the first description visible to the user in the marketplace.
-  short: Example service
-  # This description will display with additional details in the service's individual marketplace page
+  short: Peer-2-peer derivatives on Bitcoin
   long: |
-    Hello World is a bare-bones service that launches a web interface to say "Hello World", and nothing more.
-# These assets are static files necessary for packaging the service for Start9 (into an s9pk). Each value is a path to the specified asset. If an asset is missing from this list, or otherwise denoted, it will be defaulted to the values denoted below. 
+    ItchySats is a non-custodial trading solutions based on DLCs (discreet log contracts).
+    You can trade CFDs (contracts for difference) using an on-chain protocol. No counterparty risk. Just you and your keys.
 assets:
-  # Default = LICENSE.md
   license: LICENSE
-  # Default = icon.png (.svg allowed)
   icon: icon.png
-  # Default = INSTRUCTIONS.md
   instructions: instructions.md
-  # Default = image.tar
   docker-images: image.tar
-# The main action for initializing the service. Currently, the only type of action available is docker.
 main:
-  # Docker is currently the only action implementation
   type: docker
-  # Identifier for the main image volume, which will be used when other actions need to mount to this volume.
   image: main
-  # The executable binary for starting the initialization action. For docker actions, this is typically a "docker_entrypoint.sh" file. See the Dockerfile and the docker_entrypoint.sh in this project for additional details.
   entrypoint: "docker_entrypoint.sh"
-  # Any arguments that should be passed into the entrypoint executable 
   args: []
-  # Specifies where to mount the data volume(s), if there are any. Mounts for pointer dependency volumes are also denoted here. These are necessary if data needs to be read from / written to these volumes. 
   mounts:
-    # Specifies where on the service's file system its persistence directory should be mounted prior to service startup
     main: /root
-# Health checks 
-health-checks: {}
-config: ~
-properties: ~
-#  type: script
-# This denotes any data, asset, or pointer volumes that should be connected when the "docker run" command is invoked
+    data: /data
+health-checks:
```

Comment:
❓ How do you think this health checks might interact with a bunch of CFDs that take a long time to load? When are these checks run?

Reply:
They go against the `alive` endpoint which should be independent of loading the CFDs.